includes/wf_crm_webform_postprocess.inc

```diff
@@ -966,6 +966,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           $perm = $perm == 1 ? 2 : 1;
         }
       }
+      //initialise start_date for create action.
+      if (empty($existing) !array_key_exists('start_date', $params) {
```

Comment:
seems like it got messed up while pushing, was working fine while testing. I've tested the flow again and confirmed that it creates a new relationship when an expired/inactive duplicate relationship exists for a contact.

Thanks for spotting!